integration-tests/api/helpers/call-helpers.js

```diff
@@ -0,0 +1,95 @@
+const { useApi } = require("../../helpers/use-api");
+
+const header = {
+  headers: {
+    authorization: "Bearer test_token",
+  },
+};
+
+const resolveCall = async (path, payload, header) => {
```

Comment:
Method used internally here to allow us to look at the status of a failure and success (from the axios call) the same way